flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml

```diff
@@ -164,11 +172,6 @@ under the License.
 									<pattern>org.apache.parquet</pattern>
 									<shadedPattern>org.apache.hive.shaded.parquet</shadedPattern>
 								</relocation>
-								<!-- Guava dependencies that still end up in the shaded JAR are relocated to avoid clashes. -->
-								<relocation>
-									<pattern>com.google</pattern>
-									<shadedPattern>org.apache.flink.hive.shaded.com.google</shadedPattern>
-								</relocation>
```

Comment:
This is needed, I can now see we are including 2x versions of Guava into the jar:
- The non-relocated version (com.google)
- The version coming from avro (avro.shaded)

```
unzip -l target/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT.jar | grep /Futures.class
    25448  10-18-2022 20:27   com/google/common/util/concurrent/Futures.class
    19380  10-18-2022 20:27   avro/shaded/com/google/common/util/concurrent/Futures.class
```

Reply:
reverted the changes.



 ```
 unzip -l target/flink-sql-connector-hive-3.1.3_2.12-1.17-SNAPSHOT.jar | grep /Futures.class     
    31886  10-19-2022 14:36   org/apache/flink/hive/shaded/com/google/common/util/concurrent/Futures.class
    19380  10-19-2022 14:36   avro/shaded/com/google/common/util/concurrent/Futures.class
```